app/views/casa_cases/show.html.erb

```diff
@@ -95,7 +98,7 @@
 
     <div>
       <h6><strong><%= t(".assigned_volunteers") %>:</strong></h6>
-      <% policy_scope(@casa_case.assigned_volunteers).each_with_index do |volunteer, index| %>
+      <% @casa_case.assigned_volunteers.each do |volunteer| %>
```

Comment:
bring back policy :)

Reply:
that wasn't supposed to be commited, will undo